README.md

```diff
@@ -17,6 +17,13 @@ One of the reasons for developing this custom module is to feed JSON logs to log
 * Tracepoints compatibility to measure request, response times and such.
 
 # Changes
+## 2.1.0
+* Added
+	- Dependencies for mule's cryptography and compression modules
+	- Implementation to compress, encrypt, both or neither to the logger operation
+	- Unit tests for operations class
+* Updated
+	- Custom Logger class log method to optionally log the encrypted, compressed or raw payload string
```

Comment:
Could we remove Dependencies and Unit Tests from here, this should just include the features added, not related implementation details.

Reply:
Done